gradle/project.gradle.kts

```diff
@@ -73,7 +78,7 @@ allprojects {
     }
 
     tasks.withType<KotlinCompile> {
-        kotlinOptions.jvmTarget = "1.8"
+        kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
++, but why the hell is toString()? Blashpemy! Isnt there some name() or whatever() method for this?

Reply:
Disheartening